website/content/docs/connect/config-entries/jwt-provider.mdx

```diff
@@ -126,8 +150,27 @@ JSONWebKeySet = {
               MaxInterval = "10s"
             }
           }
+          JWKSCluster = {
+            DiscoveryType = "STATIC"
+            ConnectTimeout = "10s"
+            TLSCertificates = {
+              TrustedCA = {                                # cannot specify with JWKSCluster{}.CaCertificateProviderInstance
+                Filename = "<path/to/cert/file>"           # cannot specify with JWKSCluster{}.TrustedCA.EnvironmentVariable, JWKSCluster{}.TrustedCA.InlineString or JWKSCluster{}.TrustedCA.InlineBytes
+                EnvironmentVariable = "<env-variable>"     # cannot specify with JWKSCluster{}.TrustedCA.Filename, JWKSCluster{}.TrustedCA.InlineString or JWKSCluster{}.TrustedCA.InlineBytes
+                InlineString = "<inline-string>"           # cannot specify with JWKSCluster{}.TrustedCA.Filename, JWKSCluster{}.TrustedCA.EnvironmentVariable or JWKSCluster{}.TrustedCA.InlineBytes
+                InlineBytes = "\302\000\302\302\302\302"   # cannot specify with JWKSCluster{}.TrustedCA.Filename, JWKSCluster{}.TrustedCA.InlineString or JWKSCluster{}.TrustedCA.EnvironmentVariable
```

Comment:
I'm finding these comments a little confusing and long to follow. Maybe a solution would be to provide one example and in the comments we can have something like 
```
Filename = "<path/to/cert/file>"           # alternatives: EnvironmentVariable, InlineString, or InlineBytes
```

Reply:
this is also another pattern that the docs team seems to be following, I am hesitant to change it. I do agree here is looks a bit weird but in the actual docs, it doesn't look too bad
<img width="724" alt="Screen Shot 2023-07-04 at 3 49 18 PM" src="https://github.com/hashicorp/consul/assets/7438896/b4b574f9-60d8-4819-88e4-03a496404445">